examples/input_specification.txt

```diff
@@ -0,0 +1,22 @@
+output_image_path

```

Comment:
This whitespace-significant input format is unnecessarily rigid and cryptic. Switching to JSON or YAML would make input files readable and allow you to delete all the custom parsing.